src/main/java/roomescape/reservationslot/application/ReservationSlotApplicationService.java

```diff
@@ -0,0 +1,75 @@
+import roomescape.member.domain.service.MemberDomainService;
+import roomescape.reservationslot.presentation.dto.response.MyReservationSlotResponse;
+import roomescape.reservationslot.presentation.dto.response.ReservationSlotResponse;
+import roomescape.reservationtime.domain.ReservationTime;
+import roomescape.reservationtime.domain.service.ReservationTimeDomainService;
+import roomescape.theme.domain.Theme;
+import roomescape.theme.domain.service.ThemeDomainService;
+import roomescape.reservation.domain.service.ReservationDomainService;
+
+@Service
+public class ReservationSlotApplicationService {
+
+    private final ReservationSlotDomainService reservationSlotDomainService;
+    private final ReservationTimeDomainService reservationTimeDomainService;
+    private final ThemeDomainService themeDomainService;
+    private final MemberDomainService memberDomainService;
+    private final ReservationDomainService reservationDomainService;
+
+    public ReservationSlotApplicationService(final ReservationSlotDomainService reservationSlotDomainService,
+                                             final ReservationTimeDomainService reservationTimeDomainService,
+                                             final ThemeDomainService themeDomainService,
+                                             final MemberDomainService memberDomainService,
+                                             final ReservationDomainService slotReservationDomainService) {
+        this.reservationSlotDomainService = reservationSlotDomainService;
+        this.reservationTimeDomainService = reservationTimeDomainService;
+        this.themeDomainService = themeDomainService;
+        this.memberDomainService = memberDomainService;
+        this.reservationDomainService = slotReservationDomainService;
+    }
+
+    public List<ReservationSlotResponse> findReservations(final Long themeId, final Long memberId,
+                                                          final LocalDate startDate,
+                                                          final LocalDate endDate) {
+        List<ReservationSlot> filteredReservations = reservationSlotDomainService.findFilteredReservations(themeId, memberId,
+                startDate, endDate);
+        return filteredReservations
+                .stream()
+                .map(reservation -> {
+                    ReservationTime time = reservation.getTime();
+                    Theme theme = reservation.getTheme();
+                    Member member = reservation.findReservedMember();
+                    return ReservationSlotResponse.of(reservation, time, theme, member);
+                })
+                .toList();
+    }
+
+    public void delete(Long id) {
+        reservationSlotDomainService.delete(id);
+    }
+
+    public ReservationSlotResponse create(final LocalDate date, final Long timeId, final Long themeId, final Long memberId,
+                                          final LocalDateTime now) {
```

Comment:
```suggestion
                                          final LocalDateTime reservationDateTime) {
```
now 보다는 예약 일시라는 의미의 필드명이 더 적절해보여요.

Reply:
더 적합한 필드명이네요! 수정하였습니다.
[refactor: now -> reservationDateTime으로 변경](https://github.com/Starlight258/spring-roomescape-waiting/commit/0b9c8d222034c1b16edf2c26d1221fbf847b349e)